invenio_app_ils/ui/backoffice/src/pages/ItemDetails/components/ItemPendingLoans/state/__tests__/reducer.js

```diff
@@ -0,0 +1,68 @@
+import reducer, { initialState } from '../reducer';
```

Comment:
If we stick with initializing the mocked store with the initialState of our reducer code instead of defining it in the test we should probably do it everywhere in our code base. I was questioning myself about that when I was writing the tests and I had doubts.

Reply:
It is already the case boss!